src/UIState/MainMenu.cpp

```diff
@@ -170,15 +170,12 @@ void MainMenu::selectView() {
 
 void MainMenu::selectSet() {
   switch (level2) {
-    case SET_CALIBRATION_1:
-      this->setNextState((UIState *)new PHCalibration(tc));
-      break;
-    case SET_CALIBRATION_2:
-      this->setNextState((UIState *)new PHCalibration(tc));
-      break;
-    case SET_CALIBRATION_CLEAR:
-      this->setNextState((UIState *)new PHCalibration(tc));
+    case SET_CALIBRATION:
+      this->setNextState((UIState *)new PHCalibrationMid(tc));
       break;
+    // case SET_CALIBRATION_CLEAR:
```

Comment:
Since this is part of MainMenu we should probably implement at least a stub that does nothing.

Reply:
See #96.